### PR TITLE
feat: added gamut-illustrations package

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
### Overview

Added new `@codecademy/gamut-illustrations` package with `BrowserLock` and `NumberBlocks` illustrations to start.

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/8YhsJUVnCmlE2kJZE7PQJj/Account-Security?node-id=322%3A0
- [x] Related to JIRA ticket: WEB-1199
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

Screenshot of the NumberBlocks illustration in Storybook

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.3-alpha.7f3346.0<br/>@codecademy/gamut-illustrations@0.2.0-alpha.7f3346.0<br/>@codecademy/gamut-labs@7.3.1-alpha.7f3346.0<br/>@codecademy/gamut@19.2.1-alpha.7f3346.0<br/>@codecademy/styleguide@21.3.0-alpha.7f3346.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fd3ad87be726325f741d8bf--gamut-preview.netlify.app>https://5fd3ad87be726325f741d8bf--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fd3ad87be726325f741d8bf>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.3-alpha.b8cd6e.0<br/>@codecademy/gamut-illustrations@0.2.0-alpha.b8cd6e.0<br/>@codecademy/gamut-labs@7.3.1-alpha.b8cd6e.0<br/>@codecademy/gamut@19.2.1-alpha.b8cd6e.0<br/>@codecademy/styleguide@21.3.0-alpha.b8cd6e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fd3b02e0bdcbe283f34421e--gamut-preview.netlify.app>https://5fd3b02e0bdcbe283f34421e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fd3b02e0bdcbe283f34421e>Deploy Logs</a></p>

File: packages/gamut-illustrations/babel.config.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  presets: ['codecademy', '@babel/preset-typescript'],
+  plugins: [
+    'react-anonymous-display-name',
```
Comment (jakemhiller) [inline packages/gamut-illustrations/babel.config.js:4]: This is in the Babel preset now, I guess I forgot to remove it from this repo 😅

File: packages/gamut-illustrations/src/NumberBlocks.tsx [hunk 2]
```diff
@@ -0,0 +1,312 @@
+import React from 'react';
+import { IllustrationProps } from './types';
+
+export const NumberBlocks: React.FC<IllustrationProps> = ({
+  className,
+  height,
+  width,
+}) => (
+  <svg
```
Comment (jakemhiller) [inline packages/gamut-illustrations/src/NumberBlocks.tsx:9]: Did you minify these w/ svgo before putting them in here? looks kinda big (but it is a complex illustration)

File: packages/gamut-illustrations/babel.config.js (hunk 1 above)
Comment (JoshuaKGoldberg) [reply packages/gamut-illustrations/babel.config.js:4]: Oh ha right I forgot about that! Will remove here & send a PR to remove elsewhere...

...and I forgot I wanted to move Babel into this repo, will do that too.

File: packages/gamut-illustrations/src/NumberBlocks.tsx (hunk 2 above)
Comment (JoshuaKGoldberg) [reply packages/gamut-illustrations/src/NumberBlocks.tsx:9]: No, but that's a good idea. I'll try that out and see if it changes much; if it does I'll add it to the instructions.

Comment (jakemhiller) [reply packages/gamut-illustrations/src/NumberBlocks.tsx:9]: i assume we need more access to the child elements of the SVG, but if not something like this babel plugin might be useful for this: https://github.com/airbnb/babel-plugin-inline-react-svg

File: packages/styleguide/stories/Core/Illustrations/Illustrations.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+## Sandbox
+
+<Preview>
+  <Story name="Editable Illustration">
+    {(args) => {
+      const illustrationName = select(
+        'Illustration Name',
+        Object.keys(illustrations),
+        'NumberBlocks'
+      );
```
Comment (codecaaron) [inline packages/styleguide/stories/Core/Illustrations/Illustrations.stories.mdx:49]: I know this isn't specifically a prop but we could visually separate this in the ArgsTable to make it work with controls nicely :)

```tsx
<Meta 
   // Other meta props
  argTypes={{ 
     name: 'Illustration Key',
     description:  'This is the import name of the illustration from the package and not a true prop',
     table: {
         category: 'Story Options',
      },
     control: {
         type: 'select',
         options: Object.keys(illustrations),
     },
  }}
/>
```

File: packages/gamut-illustrations/src/NumberBlocks.tsx (hunk 2 above)
Comment (JoshuaKGoldberg) [reply packages/gamut-illustrations/src/NumberBlocks.tsx:9]: Yeah we will - e.g. for color variations.

Comment (JoshuaKGoldberg) [reply packages/gamut-illustrations/src/NumberBlocks.tsx:9]: TWIL about https://jakearchibald.github.io/svgomg, what a great tool...!

Comment (JoshuaKGoldberg) [reply packages/gamut-illustrations/src/NumberBlocks.tsx:9]: Oh, hang on, some of the changes are destructive... tinkering now. 
Poorly cropped illustration image

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.3-alpha.11949e.0<br/>@codecademy/gamut-illustrations@0.2.0-alpha.11949e.0<br/>@codecademy/gamut-labs@7.3.1-alpha.11949e.0<br/>@codecademy/gamut@19.2.1-alpha.11949e.0<br/>@codecademy/styleguide@21.3.0-alpha.11949e.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.3-alpha.0449f7.0<br/>@codecademy/gamut-illustrations@0.2.0-alpha.0449f7.0<br/>@codecademy/gamut-labs@7.3.1-alpha.0449f7.0<br/>@codecademy/gamut@19.2.1-alpha.0449f7.0<br/>@codecademy/styleguide@21.3.0-alpha.0449f7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fd3dae5a46250592c56619d--gamut-preview.netlify.app>https://5fd3dae5a46250592c56619d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fd3dae5a46250592c56619d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.3-alpha.154571.0<br/>@codecademy/gamut-illustrations@0.2.0-alpha.154571.0<br/>@codecademy/gamut-labs@7.3.1-alpha.154571.0<br/>@codecademy/gamut@19.2.1-alpha.154571.0<br/>@codecademy/styleguide@21.3.0-alpha.154571.0<br/></pre>

**Review by jakemhiller — APPROVED** [review]
lgtm!

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fd3ee9f260f0587bd30a5ed--gamut-preview.netlify.app>https://5fd3ee9f260f0587bd30a5ed--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fd3ee9f260f0587bd30a5ed>Deploy Logs</a></p>

**Review by codecaaron — APPROVED** [review]
:shipit:

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.3-alpha.2d9dad.0<br/>@codecademy/gamut-illustrations@0.2.0-alpha.2d9dad.0<br/>@codecademy/gamut-labs@7.4.1-alpha.2d9dad.0<br/>@codecademy/gamut@19.3.1-alpha.2d9dad.0<br/>@codecademy/styleguide@21.4.0-alpha.2d9dad.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.3-alpha.478076.0<br/>@codecademy/gamut-illustrations@0.2.0-alpha.478076.0<br/>@codecademy/gamut-labs@7.4.1-alpha.478076.0<br/>@codecademy/gamut@19.3.1-alpha.478076.0<br/>@codecademy/styleguide@21.4.0-alpha.478076.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fd3f2332e31766a41612a8b--gamut-preview.netlify.app>https://5fd3f2332e31766a41612a8b--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fd3f2332e31766a41612a8b>Deploy Logs</a></p>

**Pull request merged**